Adding useful info (#415)
Macrometacorp/docs: docs/sdks/install-sdks.md

</TabItem>
</Tabs>

> ## View currently installed version of the SDK

Reviewer: ```suggestion
## Check Installed SDK Version
```

> To know the currently installed version of the SDK, run the following command

Reviewer: ```suggestion
To check which version of the SDK is currently installed, run the following command in your terminal:
```

```

</TabItem>
</Tabs>

## Reference

You can follow the following links for further reference regarding the SDK's:

> jsC8: [https://www.npmjs.com/package/jsc8](https://www.npmjs.com/package/jsc8)

Reviewer: ```suggestion
- jsC8: [https://www.npmjs.com/package/jsc8](https://www.npmjs.com/package/jsc8)
```

Reviewer: Please change the links for both of these to the GitHub repos.

> jsC8: [https://github.com/macrometacorp/jsC8](https://github.com/macrometacorp/jsC8)

Reviewer: ```suggestion
- jsC8: [https://github.com/macrometacorp/jsC8](https://github.com/macrometacorp/jsC8)
```

> pyC8: [https://pyc8.readthedocs.io/en/latest](https://pyc8.readthedocs.io/en/latest)

Reviewer: ```suggestion
- pyC8: [https://pyc8.readthedocs.io/en/latest](https://pyc8.readthedocs.io/en/latest)
```

> pyC8: [https://github.com/Macrometacorp/pyC8](https://github.com/Macrometacorp/pyC8)

Reviewer: ```suggestion
- pyC8: [https://github.com/Macrometacorp/pyC8](https://github.com/Macrometacorp/pyC8)
```
